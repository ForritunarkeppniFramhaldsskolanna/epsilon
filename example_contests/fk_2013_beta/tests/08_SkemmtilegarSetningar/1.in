6
Hvernig gengur?
Forritunarkeppnin er awesome.
Was she early?
Abba Abba abba.
Abba babba babb.
John needs sugar!
