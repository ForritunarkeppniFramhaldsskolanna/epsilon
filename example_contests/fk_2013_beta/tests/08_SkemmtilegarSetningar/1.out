Fun
Boring
Fun
Fun
Boring
Fun
